doc/content/getting-started/programming-software/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Programming Software"
+description: ""
+weight:
+distributions: null
+---
+
+Become familiar with programming your executables to your {{% gn %}} with a ST-LINK programmer.
+
+<!--more-->
+
+## ST-LINK
+
+A ST-LINK is required to program the {{% gn %}}. Multiple versions exist, but most should have a similar pin-out and functionality. A [ST-LINK/V3SET](https://www.st.com/en/development-tools/stlink-v3set.html) is a good example. Most versions will show that it is programming by blinking its LEDs during the process.
+
+## Wiring your ST-LINK to the {{% gn %}}
+
+The wiring from the ST-LINK/V3 programmer &rarr; {{% gn %}} is as follows:
+
+1. VCC &rarr; VCC
+2. GND &rarr; GND
+3. CLK &rarr; CLK
+4. DIO &rarr; SWD
+5. NRST &rarr; NRST
+
+And optionally to work with UART, either for debugging or programming, you should connect:
+
+6. RX &rarr; TX
+7. TX &rarr; RX
+
+{{< note >}} Do not confuse VCC with VIN on the {{% gn %}}. VIN should still be powered separately, either with the pin or the battery connector, as it powers the {{% gn %}}. VCC is used to power the BOOT0 pin. {{</ note >}}
```

Comment:
I don't think this is correct!, VCC is target voltage indicator
![image](https://user-images.githubusercontent.com/24204263/111759284-b43be800-889d-11eb-90e3-3aeaa014ea35.png)
see
https://www.st.com/content/ccc/resource/technical/document/user_manual/group1/99/49/91/b6/b2/3a/46/e5/DM00526767/files/DM00526767.pdf/jcr:content/translations/en.DM00526767.pdf